app/src/main/java/com/example/cocktailmaster/ui/home/HomeContract.kt

```diff
@@ -8,10 +8,12 @@ interface HomeContract {
     interface Presenter: BasePresenter {
         fun getRandomDrinks()
         fun getAllAlphabets()
+        fun getSearchDrinks()
     }
 
     interface View: BaseView {
         fun loadRandomDrinks(drinks: List<Drink>)
         fun loadAlphabets(alphabets: List<Char>)
+        fun loadDrinksSearch(drinks: List<Drink>)
```

Comment:
https://github.com/android/architecture-samples/blob/todo-mvp-kotlin/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailContract.kt
View chỉ chịu trách nhiệm hiển thị dữ liệu được đưa trong param, nó k có nhiệm vụ load dữ liệu gì cả vì View không giao tiếp với Model 
Đặt tên lại cho đúng, mẫu Contract ở link 
```suggestion
        fun showRandomDrinks(drinks: List<Drink>)
        fun updateAlphabets(alphabets: List<Char>)
        fun notifyDrinksSearch(drinks: List<Drink>)
```